coordinator/src/test/scala/filodb.coordinator/queryplanner/LogicalPlanParserSpec.scala

```diff
@@ -54,7 +54,7 @@ class LogicalPlanParserSpec extends AnyFunSpec with Matchers {
     parseAndAssertResult("""http_requests_total::sum{job="app"}""")
     parseAndAssertResult("""topk(2.0,http_requests_total{job="app"})""")
     parseAndAssertResult("""quantile(0.2,http_requests_total{job="app"})""")
-    parseAndAssertResult("""count_values("freq",http_requests_total{job="app"})""")
+    parseAndAssertResult("""count_values(freq,http_requests_total{job="app"})""")
     parseAndAssertResult("""timestamp(http_requests_total{job="app"})""")
```

Comment:
First parameter in count_values is supposed to be in quotes

Reply:
Fixed.